src/helpers/array.helper.ts

```diff
@@ -10,6 +10,9 @@ export class ArrayHelper {
     }
     return array1.map((v, index: number) => [v, array2[index]]);
   }
+  public static arrayOfCount(count: number, element?: any) {
+    return Array.from(Array(count)).map((x) => element | 0);
```

Comment:
Ten pipe nie może zostać
![image](https://user-images.githubusercontent.com/21079319/100153862-05fed380-2ea5-11eb-8e95-224b64cc5536.png)

Lepiej użyć ??, bo typescript to lepiej obsługuje
![image](https://user-images.githubusercontent.com/21079319/100154030-49594200-2ea5-11eb-8f28-423ef1d7b428.png)